data/layout/telescope_positions-North-4LST.ecsv

```diff
@@ -1,17 +1,12 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
 # - {name: telescope_name, datatype: string}
 # - {name: pos_x, unit: m, datatype: float64}
 # - {name: pos_y, unit: m, datatype: float64}
 # - {name: pos_z, unit: m, datatype: float64}
-# - {name: lat, unit: deg, datatype: float64}
-# - {name: lon, unit: deg, datatype: float64}
-# - {name: utm_east, unit: m, datatype: float64}
-# - {name: utm_north, unit: m, datatype: float64}
-# - {name: alt, unit: m, datatype: float64}
 # meta:
-#   EPSG: 32628
+#   EPSG: 32719
 #   center_alt: !astropy.units.Quantity
 #     unit: &id001 !astropy.units.Unit {unit: m}
 #     value: 2150.0
```

Comment:
Strong reason not to use the most recent prod6 values here?

Reply:
I could update the values too. Should I get them from the DB @orelgueta ?

Reply:
Yes, you can find them in the DB for Prod5 (same value as Prod6). Checking the DB, the values are
- North - 32628
- South - 32719

Which means the issue here is that the file was changed from North EPSG to South EPSG which is probably not what we want.